### ISSUE TITLE
random line shown as bold, but shouldn't

### ISSUE THREAD

**Issue by timotheecour** [nim-lang/nimforum#178]
why is the following line shown as bold here: https://forum.nim-lang.org/t/4037#25141

![image](https://user-images.githubusercontent.com/2194784/42663760-d8ef2b9c-85eb-11e8-8305-e7d7d199492a.png)

```
note: not sure why line below is bold, RST styling seems wrong, nothing I wrote indicates it should be bold; @dom96 is that a bug?

Reusing `typed` in that context is very confusing (as this above discussion proves) and, in the current 
 implementation which means `void`, has completely different meaning from `typed` for parameters.

in conclusion: either output `typed` means void (and void should be used instead), or implementation should be changed and output `typed` should mean `auto` (and auto should be used instead).
```
